src/Microsoft.Azure.SignalR.AspNet/HubHost/ServiceMessageBus.cs

```diff
@@ -17,14 +18,16 @@ internal class ServiceMessageBus : MessageBus
         private readonly IServiceConnectionManager _serviceConnectionManager;
         private readonly IClientConnectionManager _clientConnectionManager;
         private readonly IAckHandler _ackHandler;
+        private readonly ILogger<ServiceMessageBus> _logger;
 
-        public ServiceMessageBus(IDependencyResolver resolver) : base(resolver)
+        public ServiceMessageBus(IDependencyResolver resolver, ILogger<ServiceMessageBus> logger) : base(resolver)
         {
             // TODO: find a more decent way instead of DI, it can be easily overriden
             _serviceConnectionManager = resolver.Resolve<IServiceConnectionManager>() ?? throw new ArgumentNullException(nameof(IServiceConnectionManager));
             _clientConnectionManager = resolver.Resolve<IClientConnectionManager>() ?? throw new ArgumentNullException(nameof(IClientConnectionManager));
             _parser = resolver.Resolve<IMessageParser>() ?? throw new ArgumentNullException(nameof(IMessageParser));
             _ackHandler = resolver.Resolve<IAckHandler>() ?? throw new ArgumentNullException(nameof(IAckHandler));
+            _logger = logger ?? throw new ArgumentNullException(nameof(ILogger<ServiceMessageBus>));
```

Comment:
`nameof(logger)`

Reply:
I just keep the conventions... look at the previous lines from L26-L29